bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -196,7 +196,9 @@ <h2 class="section-headline">{{ _('Our innovations') }}</h2>
         <ul id="icon-links-innovate" class="icon-links">
         {% if l10n_has_tag('home-update-20171116') %}
           <li id="common-voice">
-            <a rel="external" href="https://voice.mozilla.org/"><svg class="icon"><use xlink:href="#icon-common-voice"></use></svg>{{ _('Common Voice') }}</a>
+            {% trans url='https://voice.mozilla.cz/' if LANG == 'cs' else 'https://voice.mozilla.org/' %}
```

Comment:
There's no need to wrap the HTML in a `trans` block here, since this changes the string itself (and invalidates the existing translations). I'd recommend using a variable e.g. 

```
{% set voice_url = 'https://voice.mozilla.cz/' if LANG == 'cs' else 'https://voice.mozilla.org/' %}
```

You can then use the variable via `<a rel="external" href="{{ voice_url }}">`.

Same applies to the occurences below.